Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPostCampaignCheckoutViewController.swift

```diff
@@ -306,6 +306,21 @@ final class NoShippingPostCampaignCheckoutViewController: UIViewController,
         let thanksVC = ThanksViewController.configured(with: thanksPageData)
         self?.navigationController?.pushViewController(thanksVC, animated: true)
       }
+
+    self.viewModel.outputs.checkoutError
+      .observeForUI()
+      .observeValues { [weak self] error in
+
+        #if DEBUG
+          let serverError = error.errorMessages.first ?? ""
```

Comment:
Nit: Maybe put a default value (`Unknown server error` or something) instead of an empty string here, since this is for debugging